demo/spec/request-review-takeover.html

```
@@ -0,0 +1,82 @@
+            </div>
+          </article>
+          <article class="spec-panel">
+            <h2>Takeover rules</h2>
+            <ul>
+              <li>Takeover records direct human control over the affected scope.</li>
+              <li>Takeover states are <code>requested</code>, <code>locked</code>, <code>human_active</code>, <code>reconciliation_required</code>, <code>resumed</code>, and <code>closed</code>.</li>
+              <li>Takeover uses <code>lock_epoch</code> to reject stale autonomous continuation.</li>
+              <li>Resumed Takeover requires <code>reconciliation_refs</code>, <code>resumed_by_actor_id</code>, and <code>resolved_at</code>.</li>
+              <li>The protocol rejects stale Takeover continuation and missing Takeover reconciliation.</li>
+            </ul>
+            <p>Allowed transitions are <code>requested -> locked</code>, <code>requested -> closed</code>, <code>locked -> human_active</code>, <code>locked -> reconciliation_required</code>, <code>locked -> closed</code>, <code>human_active -> reconciliation_required</code>, <code>human_active -> closed</code>, <code>reconciliation_required -> resumed</code>, <code>reconciliation_required -> closed</code>, and <code>resumed -> closed</code>.</p>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)